dotnet/src/Connectors/Connectors.UnitTests/OpenAI/ChatCompletion/AzureOpenAIChatCompletionServiceTests.cs

```diff
@@ -700,6 +700,212 @@ public async Task GetChatMessageContentsWithChatMessageContentItemCollectionAndS
         Assert.Equal("image_url", contentItems[1].GetProperty("type").GetString());
     }
 
+    [Fact]
+    public async Task FunctionCallsShouldBePropagatedToCallersViaChatMessageItemsOfTypeFunctionCallContentAsync()
```

Comment:
`FunctionCallRequestContent`?
```suggestion
    public async Task FunctionCallsShouldBePropagatedToCallersViaChatMessageItemsOfTypeFunctionCallRequestContentAsync()
```